ports/atmel-samd/common-hal/busio/SPI.c

```diff
@@ -98,8 +98,8 @@ void common_hal_busio_spi_construct(busio_spi_obj_t *self,
     uint8_t miso_pad = 0;
     uint8_t dopo = 255;
 
-    // Special case for SAMR boards.
-    #ifdef PIN_PC19
+    // Special case for SAMR21 boards. (feather_radiofruit_zigbee)
+    #if defined(PIN_PC19F_SERCOM4_PAD0)
```

Comment:
Is it worth checking for SAMx21 architecture here, or is that macro def unique to this particular case?

Reply:
```
circuitpython/ports/atmel-samd/asf4$ git grep -l PIN_PC19F_SERCOM4_PAD0
samd21/include/pio/samr21e16a.h
samd21/include/pio/samr21e17a.h
samd21/include/pio/samr21e18a.h
samd21/include/pio/samr21g16a.h
samd21/include/pio/samr21g17a.h
samd21/include/pio/samr21g18a.h
```
The SERCOM0 exceptional pad placement is only in the "samr" family MCU headers, so this conditional block wouldn't be entered on any SAMD/E builds. But if it did, it's OK(-ish) If Pin C19 has function F as SERCOM4 PAD0, we'd probably _want_ to engage the special case there.  We might have to check that the other exceptions match up too.

(there are other SAMR "function F" sercom pads but given that there's one SAMR board I assume the other special cases aren't important)